src/components/note/NotePageContent.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { useCreateNoteMutation } from '../../redux/slices/notesSliceApi';
+import { toast } from 'react-toastify';
+
+import ChooseColor from '../shared/ChooseColor';
+import Button from '../shared/Button';
+import TextArea from '../shared/TextArea';
+
+const NotePageContent = () => {
+  const [description, setDescription] = useState('');
+  const colorArray = ['#6074F9', '#E42B6A', '#5ABB56', '#3D3A62', '#F4CA8F'];
+  const [activeColor, setActiveColor] = useState(0);
+
+  const [createNote] = useCreateNoteMutation();
+
+  const handleCreateNote = async () => {
+    try {
+      let res = await createNote({
+        description: description,
+        color: `${colorArray[activeColor]}`,
+        owner_id: `${localStorage.getItem('user_id')}`,
```

Comment:
** Suggestion ** 
why do you keep user_id in LS, could you please keep it in store and get it by useSelector() ?